fastlane/Fastfile

```diff
@@ -245,6 +246,58 @@ platform :ios do
     )
   end
 
+  lane :generate_changelog do
+
+  end
+
+  # Inspired by fastlane's changelog
+  # https://github.com/fastlane/fastlane/blob/eb4584e0394c1fecccd70d4313260f4707f2af80/fastlane/Fastfile#L346-L380
+  desc "Generate changelog from GitHub compare and PR data for mentioning GitHub usernames in release notes"
+  lane :github_changelog do |options|
+    last_version = sh("git describe --tags --abbrev=0").strip
```

Comment:
🤔 would this work when hotfixing an older major?

Reply:
we have a lane around for getting the version number from the file called VERSION, we could use that instead

Reply:
@aboedo That didn't work because at the time its the `<next-verison>-SNAPSHOT` 😬 

The latest tag that was made was `3.X.X` something but that tag is on different branches (and not main) so this command actually found the correct one which was `4.3.0` 🙃

I was hoping to use `.version` but it was already bumped 🤷‍♂️

Reply:
oohh right that makes sense! 👍